package.json

```diff
@@ -105,6 +105,7 @@
 		"version": "npm run bundle && git add -A bundle"
 	},
 	"dependencies": {
-		"email-validator": "2.0.4"
+		"email-validator": "2.0.4",
+		"@sentry/react-native": "2.4.2"
```

Comment:
I had to use this specific version of the package because we have to match the dependency version of Sentry iOS SDK ([reference in version `2.4.2`](https://github.com/getsentry/sentry-react-native/blob/a1b314781fb673becfb572a97a297b9bb8e7a849/RNSentry.podspec#L20)) with the one we have in [`Automattic-Tracks-iOS`](https://github.com/Automattic/Automattic-Tracks-iOS/blob/36db74cceaf98e107e0c713eaae81eb4139c2059/Automattic-Tracks-iOS.podspec#L40).